spec/yaml/swiftnav/sbp/piksi.yaml

```diff
@@ -17,6 +17,7 @@ definitions:
 
  - MSG_PRINT:
     id: 0x0010
+    short_desc: Information and debugging information.
```

Comment:
"Information" repeated - maybe something like "String containing errors, warnings and informational messages"?